lib/sqlalchemy/orm/__init__.py

```diff
@@ -231,17 +231,14 @@ def clear_mappers():
     upon a fixed set of classes.
 
     """
-    mapperlib._CONFIGURE_MUTEX.acquire()
-    try:
-        while _mapper_registry:
```

Comment:
this is an old jython workaround that isn't needed, however the syntax change here removes the "while"

Reply:
yeah, this is wrong, will fix